app/assets/stylesheets/_screen.scss

```diff
@@ -625,6 +617,63 @@ div#keycut-help {
   }
 }
 
+/* Settings */
+.settings-page {
+  height: 100%;
+  padding: 0 30px;
+  overflow-y: auto;
+}
+
+/* Import */
+.import-form {
+  margin: 15px auto;
+  width: 70%;
+
+  .field {
+    &.select {
+      padding: 5px;
+      background: $lightgrey;
+      border-radius: 5px;
+      padding: 10px;
+      margin-bottom: 10px;
+    }
+    &.submit {
+      text-align: right;
+    }
+  }
+}
+
+.instructions > p {
```

Comment:
Can we avoid styling an element directly here? Isn't there a class for this `p` element?